wordpress-mobile/WordPress-iOS
# Post list revamp

Fixes #11646

This PR will merge in all changes for the post list revamp 🎉

To test:
- Because this is the result of several previous PRs, you don't have to review the entire PR (that's the hope of breaking it up!) However, as a reviewer, you're welcome to comment on anything 😀 
- Ensure that the post list matches the mockups (see #11646)
- Also ensure that any features added to the post list recently (last 1-2 months) are still present, due to this being a long-lived feature branch 😀 

**Note:** This doesn't have the new colors, as they landed out-of-order. We should fix those in a separate PR.

Update release notes:
